x-pack/plugins/security/server/authentication/authentication_service.ts

```diff
@@ -201,8 +210,14 @@ export class AuthenticationService {
         ? this.authenticator.getRequestOriginalURL(request)
         : `${http.basePath.get(request)}/`;
       if (!isLoginPageAvailable) {
+        const customBrandingValue = await customBranding.getBrandingFor(request, true);
```

Comment:
__optional nit:__ we try to use this convention for unnamed number/boolean function arguments (but usually just opt to `options: { unauthenticated: boolean }` signature to make it easier to read the code).
```suggestion
        const customBrandingValue = await customBranding.getBrandingFor(request, /* unauthenticated */ true);
```